packages/app/utilities.ts

```diff
@@ -637,24 +637,17 @@ export const findTokenChainName = (chainId?: string) => {
 
 export const getFormatDistanceStrictToWeek = (time?: string) => {
   if (!time) return "";
-  const distanceDays = formatDistanceStrict(new Date(time), new Date(), {
-    unit: "day",
-  });
-  const days = Number(distanceDays.split(" ")[0]);
-  if (days === 0) {
-    const distanceHours = formatDistanceStrict(new Date(time), new Date(), {
-      unit: "hour",
-    });
-    const hours = Number(distanceHours.split(" ")[0]);
-    return `${hours}h`;
-  }
-  if (days >= 7) {
-    return `${Math.ceil(days / 7)}w`;
+  const currentDate = new Date();
```

Comment:
Actually, we could place this outside, which would save one more `new Date` call.

Reply:
Thumbs up = it's ugly don't do it? 

Or shall I do it?

Reply:
@alantoa @intergalacticspacehighway 
Thumbs up = it's ugly don't do it?

Or shall I do it?

Reply:
I prefer to keeping inner!